aerospike/src/main/java/com/yahoo/ycsb/db/AerospikeClient.java

```diff
@@ -0,0 +1,207 @@
+package com.yahoo.ycsb.db;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+import java.util.Vector;
+
+import com.aerospike.client.AerospikeException;
+import com.aerospike.client.Bin;
+import com.aerospike.client.Key;
+import com.aerospike.client.Record;
+import com.aerospike.client.ResultCode;
+import com.aerospike.client.policy.ClientPolicy;
+import com.aerospike.client.policy.Policy;
+import com.aerospike.client.policy.RecordExistsAction;
+import com.aerospike.client.policy.WritePolicy;
+
+import com.yahoo.ycsb.ByteArrayByteIterator;
+import com.yahoo.ycsb.ByteIterator;
+import com.yahoo.ycsb.DBException;
+
+public class AerospikeClient extends com.yahoo.ycsb.DB {
+  private static final boolean DEBUG = false;
+
+  private static final String DEFAULT_HOST = "localhost";
+  private static final String DEFAULT_PORT = "3000";
+  private static final String DEFAULT_TIMEOUT = "10000";
+  private static final String DEFAULT_NAMESPACE = "ycsb";
+
+  private static final int RESULT_OK = 0;
+  private static final int RESULT_ERROR = 1;
```

Comment:
These should be unified eventually, but negative values seem more prevalent for errors in most YCSB clients - consider using -1.